runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,68 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({opts})                                         *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range. This method does not make an LSP request, but just queries locally
+    stored inlay hints. The returned inlay hints are unresolved.
+
+    Example usage: >lua
+        local timeout, bufnr = 100, vim.api.nvim_get_current_buf()
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
+
+        local client = vim.lsp.get_clients({ id = hint.client_id })[1]
+        resolved_hint = client.request_sync(
+          'inlayHint/resolve', hint.inlay_hint, timeout, bufnr).result
+        vim.lsp.util.apply_text_edits(resolved_hint.textEdits, bufnr, 'utf-8')
```

Comment:
```suggestion
        vim.lsp.util.apply_text_edits(resolved_hint.textEdits, bufnr, client.encoding)
```

utf-8 is wrong for all but a handful language servers

Reply:
Thanks for the tip!